src/cmd/dev.go

```diff
@@ -50,10 +52,61 @@ func NewDevCommand() *cobra.Command {
 	cmd.AddCommand(NewDevFindImagesCommand(v))
 	cmd.AddCommand(NewDevGenerateConfigCommand())
 	cmd.AddCommand(NewDevLintCommand(v))
+	cmd.AddCommand(NewDevInspectCommand(v))
 
 	return cmd
 }
 
+// NewDevInspectCommand creates the `dev inspect` sub-command.
+func NewDevInspectCommand(v *viper.Viper) *cobra.Command {
```

Comment:
Let's wait with exporting these for the community call later today, and based on the decision from there, make it private.